### ISSUE TITLE
Create high-level summary of the Zap Protocol with simple analogies

### ISSUE THREAD

**Issue by DanFTW** [zapproject/docs#25]
Speak in laymans terms, this is meant to be a pre-introduction of Introduction to Zap Oracles. Think of this as sort of a sales pitch to grab readers attention.

**Comment by SajidAbdurrahman** [comment]
Draft: What Zap Protocol provides what competing platforms simply do not is the much need merger between blockchain oracles and bonding curves. Bonding curves are simply algorithmic tools for predicting market parameters, namely in pricing and supply. By implementing this tool into the pricing scheme, a new feeling of predictability is ushered in, and heightened liquidity thereafter. Essentially, new players can buy-in at their desired quantity knowing the exact price, and sell at any time using the same knowledge. 

The Zap Protocol is aiming to give the consumer not only a trim and pretty hub to operate in, but a heightened sense of confidence in his investing decisions. Zap finds its merit particularly in accuracy and predictability.
